index.js

```diff
@@ -377,6 +377,22 @@ module.exports = {
     },
 
     /**
+     * Call this to enable forked type checking for TypeScript loader
+     * https://github.com/TypeStrong/ts-loader/blob/v2.3.0/README.md#faster-builds
+     *
+     * This is a build optimization API to reduce build times.
+     *
+     *
+     * @param {Object} forkedTypeScriptTypesCheckOptions
+     * @return {exports}
+     */
+    enableForkedTypeScriptTypesChecking(forkedTypeScriptTypesCheckOptions = {}) {
```

Comment:
We've made these options arguments callbacks everywhere else. Though it feels unnecessary here, I think we should do the same